### PR TITLE
Replace Demisto to Cortex XSOAR

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Replace Demisto to Cortex XSOAR

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Tanium/Integrations/Tanium_v2/README.md [hunk 1]
```diff
@@ -3720,7 +3720,7 @@
         "ExpireSeconds": 780,
         "HistorySavedQuestionId": 0,
         "ID": 19886,
-        "Name": "Trace - Install Endpoint Certificate [Windows] via Demisto API",
+        "Name": "Trace - Install Endpoint Certificate [Windows] via Cortex XSOAR API",
```
Comment (roysagi) [inline Packs/Tanium/Integrations/Tanium_v2/README.md:3723]: revert this change

File: Packs/Tanium/Integrations/Tanium_v2/README.md [hunk 2]
```diff
@@ -3775,7 +3775,7 @@
         <td> 780 </td>
         <td> 0 </td>
         <td> 19886 </td>
-        <td> Trace - Install Endpoint Certificate [Windows] via Demisto API </td>
+        <td> Trace - Install Endpoint Certificate [Windows] via Cortex XSOAR API </td>
```
Comment (roysagi) [inline Packs/Tanium/Integrations/Tanium_v2/README.md:3778]: revert this change

File: Packs/Tanium/Integrations/Tanium_v2/README.md [hunk 3]
```diff
@@ -5217,7 +5217,7 @@ There are no context output for this command.
         "ExpireSeconds": 780,
         "HistorySavedQuestionId": 0,
         "ID": 19881,
-        "Name": "Trace - Install Endpoint Certificate [Windows] via Demisto API",
+        "Name": "Trace - Install Endpoint Certificate [Windows] via Cortex XSOAR API",
```
Comment (roysagi) [inline Packs/Tanium/Integrations/Tanium_v2/README.md:5220]: revert this change

File: Packs/Tanium/Integrations/Tanium_v2/README.md [hunk 4]
```diff
@@ -5272,7 +5272,7 @@ There are no context output for this command.
         <td> 780 </td>
         <td> 0 </td>
         <td> 19881 </td>
-        <td> Trace - Install Endpoint Certificate [Windows] via Demisto API </td>
+        <td> Trace - Install Endpoint Certificate [Windows] via Cortex XSOAR API </td>
```
Comment (roysagi) [inline Packs/Tanium/Integrations/Tanium_v2/README.md:5275]: revert this change

**Review by roysagi — APPROVED** [review]
Good! see a few comments here.

File: Packs/ServiceNow/Integrations/ServiceNow/README.md [hunk 5]
```diff
@@ -3210,4 +3210,4 @@
 <h2>Troubleshooting</h2>
 <p>If there is an error within ServiceNow, the message is usually indicative, e.g., <code>User Not Authenticated, details: Required to provide Auth information </code>, <code>Invalid table</code> or <code>No Record found, details: Record doesn't exist or ACL restricts the record retrieval</code>.</p>
 <p>However there may be connection issues, e.g., when the ServiceNow server URL is incorrect or is unreachable.</p>
-<p>If you experience any issues, contact <a href="mailto:support@demisto.com">Demisto Support</a>.</p>
\ No newline at end of file
+<p>If you experience any issues, contact <a href="mailto:support@demisto.com">Cortex XSOAR Support</a>.</p>
```
Comment (yaakovi) [inline Packs/ServiceNow/Integrations/ServiceNow/README.md:3213]: i think the mail address should be replaced too

File: Packs/Tanium/Integrations/Tanium_v2/README.md (hunk 4 above)
Comment (yaakovpraisler) [reply Packs/Tanium/Integrations/Tanium_v2/README.md:5275]: done.

**Review by MislavReversingLabs — APPROVED** [review]

**Review by MislavReversingLabs — APPROVED** [review]

**Comment by MislavReversingLabs** [comment]
Is it mandatory for you to send emails for all repository actions to the email address stated in pack_metadata.json? In our case we don't want these emails sent to that address.
